Base/usr/share/man/man5/GML/Usage.md

```diff
@@ -6,43 +6,47 @@ GML Usage
 
 How to use GML in SerenityOS C++ applications
 
+## Note
+
+This manpage describes the new method of loading GML files via generated C++ code. There also is the runtime `load_from_gml` function which has the same behavior. This method should not be used except for specific use cases such as [GMLPlayground](help://man/1/Applications/GMLPlayground).
```

Comment:
Is this just me being ultra-pedantic (or wrong) with my English skills, or is the "this" in the last sentence either ambiguous or referring to the wrong loading method entirely?

Reply:
No, "This" refers to the runtime method, I don't think there's ambiguity here.